samples/csharp/common/AutoML/ConsoleHelper.cs

```diff
@@ -100,7 +100,7 @@ internal static void PrintIterationMetrics(int iteration, string trainerName, Re
 
         internal static void PrintIterationMetrics(int iteration, string trainerName, RankingMetrics metrics, double? runtimeInSeconds)
         {
-            CreateRow($"{iteration,-4} {trainerName,-35} {metrics?.NormalizedDiscountedCumulativeGains[0] ?? double.NaN,9:F4} {metrics?.NormalizedDiscountedCumulativeGains[2] ?? double.NaN,9:F4} {metrics?.NormalizedDiscountedCumulativeGains[9] ?? double.NaN,9:F4}  {metrics?.DiscountedCumulativeGains[9] ?? double.NaN,9:F4} {runtimeInSeconds.Value,9:F1}", Width);
+            CreateRow($"{iteration,-4} {trainerName,-35} {metrics?.NormalizedDiscountedCumulativeGains[0] ?? double.NaN,9:F4} {metrics?.NormalizedDiscountedCumulativeGains[2] ?? double.NaN,9:F4} {metrics?.DiscountedCumulativeGains[0] ?? double.NaN,9:F4} {runtimeInSeconds.Value,9:F1}", Width);
```

Comment:
I see you took out `NDCG@10`. Are the printed metrics too wide on the screen? 

If the space is important, the header could be reduced the from full `NormalizedDiscountedCumulativeGain@10` to `NDCG@10`. 

The AutoML code is optimizing `NDCG@10` / `DCG@10` ([comment](https://github.com/dotnet/machinelearning-samples/pull/852#discussion_r498425201)), hence it's beneficial to print that specific `@k` value.

Reply:
There were only three items. But since the third item is the 10th, I'll include that. Thanks for catching this!

Reply:
Did some improvements here and in the header.

Reply:
#Resolved